### ISSUE TITLE
Few Shot multi-label classifier

### ISSUE THREAD

**Issue by vinay1986** [flairNLP/flair#2909]
Is there an updated version of few shot multi-label classifier (trained on tars-base)?

Was referring to the article below (however did not get the desired results):

[https://towardsdatascience.com/zero-and-few-shot-learning-c08e145dc4ed](https://towardsdatascience.com/zero-and-few-shot-learning-c08e145dc4ed)

Modified the code as below:
`from flair.datasets import FlairDatapointDataset
from flair.data import Corpus,Sentence
###### training dataset consisting of four sentences (2 labeled as "olahraga" (sports) and 2 labeled as "politik" (politics)
train = FlairDatapointDataset(
    [
        Sentence('Pemilu akan dimulai bulan depan').add_label('olahraga_atau_politik', 'politik'),
        Sentence('Perdana menteri mengumumkan dana bantuan banjir besar-besaran').add_label('olahraga_atau_politik', 'politik'),
        Sentence('Olimpiade akan diadakan di Tokyo').add_label('olahraga_atau_politik', 'olahraga'),
        Sentence('Asian Games 2018 merupakan ajang multi sport yang diadakan di Jakarta').add_label('olahraga_atau_politik', 'olahraga')
    ])
###### test dataset consisting of two sentences (1 labeled as "olahraga" and 1 labeled as "politik")
test = FlairDatapointDataset(
    [
        Sentence('Para menteri kabinet memainkan permainan yang tidak menyenangkan').add_label('olahraga_atau_politik', 'politik'),
        Sentence('Ayo pergi dan bermain basket').add_label('olahraga_atau_politik', 'olahraga')
    ])
###### make a corpus with train and test split
corpus = Corpus(train=train, test=test)

from flair.trainers import ModelTrainer
from flair.models.tars_model import TARSClassifier

###### 1. load base TARS
tars = TARSClassifier.load('tars-base')
###### 2. make the model aware of the desired set of labels from the new corpus
label_dictionary=corpus.make_label_dictionary(label_type='olahraga_atau_politik')
tars.add_and_switch_to_new_task("OLAHRAGA_POLITIK", label_dictionary=label_dictionary,label_type='topic')
###### 3. initialize the text classifier trainer with your corpus
trainer = ModelTrainer(tars, corpus)

###### 4. train model
trainer.train(base_path='resources/taggers/olahraga_politik', # path to store the model artifacts
              learning_rate=0.02,
              mini_batch_size=1,
              max_epochs=20,
              shuffle=False,
              checkpoint=False,
              train_with_test=True,
              train_with_dev=True
              )
###### 5. Load few-shot TARS model
tars = TARSClassifier.load('resources/taggers/olahraga_politik/final-model.pt')

###### 6. Prepare a test sentence
sentence = Sentence("Para menteri kabinet memainkan permainan yang tidak menyenangkan")
###### 7. Predict for olahraga and politik
s1 = tars.predict(sentence)
sentence,s1

`
